From 8d0c0938c2d6c93f4b8f6ac4eb66bdb68a510f3e Mon Sep 17 00:00:00 2001
From: TDVP Build <build@tdvp.local>
Date: Fri, 14 Aug 2026 10:28:00 +0800
Subject: [PATCH] package/librsvg: install the target gdk-pixbuf SVG loader

swaybg uses gdk-pixbuf to load the desktop wallpaper.  The gdk-pixbuf cache
created by the host tooling contains the SVG loader entry, but the target
librsvg package had explicitly disabled the module.  That leaves a dangling
loader entry and causes gdk-pixbuf to fail before it can load PNG wallpapers.

Build the target loader so the cache and target filesystem agree.
---
 package/librsvg/librsvg.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/librsvg/librsvg.mk b/package/librsvg/librsvg.mk
index 523d299821..987f5d7b01 100644
--- a/package/librsvg/librsvg.mk
+++ b/package/librsvg/librsvg.mk
@@ -13,7 +13,7 @@ LIBRSVG_INSTALL_STAGING = YES
 LIBRSVG_CONF_ENV = \
 	LIBS=$(TARGET_NLS_LIBS) \
 	RUST_TARGET=$(RUSTC_TARGET_NAME)
-LIBRSVG_CONF_OPTS = --disable-pixbuf-loader --disable-tools
+LIBRSVG_CONF_OPTS = --enable-pixbuf-loader --disable-tools
 HOST_LIBRSVG_CONF_OPTS = --enable-introspection=no
 HOST_LIBRSVG_DEPENDENCIES = host-cairo host-gdk-pixbuf host-libglib2 host-libxml2 host-pango host-rustc
 LIBRSVG_LICENSE = LGPL-2.1+
-- 
2.43.0
